### DISCUSSION TITLE
Alternative Streaming

### DISCUSSION THREAD

**Discussion by ryancb4** [paullouisageneau/libdatachannel#459] [Q&A] [answered]
Hey, 
I was looking into the streaming example and it uses h264 to stream to the browser. I would like to establish a stream with images (as close to raw as possible). It seems the data-channel itself has a max packet size of 256 KB. What is the best way to use libdatachannel to stream these raw images from c++ to the browser?

**Comment by paullouisageneau** [comment, marked as answer]
If you want to stream raw images, you should not use media transport and indeed prefer Data Channels. Data Channels have a default maximum message size of 256 KB, but this can be changed by setting `rtc::Configuration::maxMessageSize` when creating the Peer Connection.
